conf/application.conf

```diff
@@ -130,7 +134,7 @@ feature-switch {
   privateBeta = true
   sessionCookieService = false
   ema-supporting-agents-enabled = true
-  isPrePopEnabled = false
```

Comment:
as this is true, need config PR's where in production this should be set to false